bing_bert/mpi_train_bert_onebitadam_bsz4k_seq128.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Can we add a comment to say how to run this and advise using the deepspeed launcher instead (for the common case)?